manager/testdata/new-blueprints/credit-card-detection-module.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: credit-card-detector
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+spec:
+  multi-user: false
+  chart:
+    name: ghcr.io/mesh-for-data/cc-detector:0.1.0
+  statusIndicators: # Indicators for a K8s deployment
+  kind: deployment # Assume for now that the program is a K8s deployment per dataset
```

Comment:
Currently, statusIndicator allow to check status of a non-standard resource that can not be computed by helm/kstatus.
IIUC it meaning Is now changed - it no longer refers to only non-standard resource but also indicates the resource type that we should look for its status - if its configuration kind then we will look in the status of the config-map. if its deployment we will look at the status of the deployment (possibly with the same name of the deployed m4dmodule). is my understanding correct? Thanks

Reply:
It did not change. It's the same as currently. I'm just writing with the comment that this is probably a K8s deployment type. (same as in current arrow flight service module that does not have any statusIndicator at all)